packages/cloud_firestore/cloud_firestore/macos/Classes/FLTCloudFirestorePlugin.m

```diff
@@ -0,0 +1 @@
+../../darwin/Classes/FLTCloudFirestorePlugin.m
```

Comment:
Are these symlinks?

Reply:
Yeah, so that we can share macOS and iOS code